spaceship/lib/spaceship/du/du_client.rb

```diff
@@ -99,61 +100,15 @@ def upload_file(app_version: nil, upload_file: nil, path: nil, content_provider_
     # You can find this by uploading an image in App Store Connect
     # then look for the X-Apple-Upload-Validation-RuleSets value
     def picture_type_map
-      # rubocop:enable Layout/ExtraSpacing
-      {
-        ipad:         "MZPFT.SortedTabletScreenShot",
-        ipad105:      "MZPFT.SortedJ207ScreenShot",
-        ipadPro:      "MZPFT.SortedJ99ScreenShot",
-        ipadPro11:    "MZPFT.SortedJ317ScreenShot",
-        ipadPro129:   "MZPFT.SortedJ320ScreenShot",
-        iphone35:     "MZPFT.SortedScreenShot",
-        iphone4:      "MZPFT.SortedN41ScreenShot",
-        iphone6:      "MZPFT.SortedN61ScreenShot",
-        iphone6Plus:  "MZPFT.SortedN56ScreenShot",
-        iphone58:     "MZPFT.SortedD22ScreenShot",
-        iphone65:     "MZPFT.SortedD33ScreenShot",
-        watch:        "MZPFT.SortedN27ScreenShot",
-        watchSeries4: "MZPFT.SortedN131ScreenShot",
-        appleTV:      "MZPFT.SortedATVScreenShot",
-        desktop:      "MZPFT.SortedDesktopScreenShot"
-      }
+      Spaceship::Tunes::DeviceType.all.map { |k, v| [k, v.picture_type] }.to_h
```

Comment:
Maybe we should keep a few of the old lines here as a comment to understand what is going to be returned from these methods? (this one and below)

Reply:
`Array#to_h` is introduced since Ruby 2.1. Unfortunately, our TargetRubyVersion in rubocop is still 2.0 😭 
Is this OK? 

https://ruby-doc.org/core-2.1.0/Array.html#method-i-to_h

Reply:
Hm, good question - if CI doesn't complain we should be good, right?

Reply:
Currently, we runs tests only above Ruby 2.3. That is why CI doesn't complain.
To be exact, we should write in Ruby 2.0 compatible code because rubocop currently supports Ruby 2.0.
However, that is tough. So we don't care this 😛

Reply:
I expected rubocop to complain on CI when we use stuff we shouldn't. Not?

Reply:
rubocop is static analyzer so they can't detect whether a method is defined or not.
This is why rubocop said nothing.
But this code doesn't work on old Ruby at runtime.
There is an only way to notice incompatible is running CI also on old Ruby versions.

In my opinion, currently, there are no CI environment for Ruby 2.0, so we should not care about old Ruby and we should update rubocop ASAP.

Reply:
Ok, understood. Best create an issue so we can discuss this and see if there are reasons against it.